component/src/test/java/org/wso2/extension/siddhi/io/websocket/sink/util/WebSocketReceiver.java

```diff
@@ -0,0 +1,54 @@
+/*
+ *  Copyright (c) 2017 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Missing comma, please correct in all places.

Reply:
fixed in 9fea7d3